index.html

```diff
@@ -88,7 +95,9 @@ <h2>Interactive Gantt Chart entirely made in SVG!</h2>
 			},
 			on_view_change: function(mode) {
 				console.log(mode);
-			}
+			},
+			view_mode: 'Month',
+			language: 'ru'
```

Comment:
Good to show off functionality, but having the example be Russian could confuse people